sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -35,3 +36,47 @@ case class Size(child: Expression) extends UnaryExpression with ExpectsInputType
     nullSafeCodeGen(ctx, ev, c => s"${ev.primitive} = ($c).size();")
   }
 }
+
+case class ArrayContains(left: Expression, right: Expression) extends BinaryExpression with ExpectsInputTypes {
+  override def dataType: DataType = BooleanType
+
+  override def inputTypes: Seq[AbstractDataType] = Seq(ArrayType, AnyDataType)
```

Comment:
Overriding the `checkInputDataTypes()` will provide more readable message once the data type is not as the expected, otherwise, it probably throws exception in the code snippet that I showed above.